app-i18n/uchardet/uchardet-0.0.6-r2.ebuild

```diff
@@ -1,13 +1,19 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
This'd require a revbump and dropping KEYWORDS to ~.

I don't really see a reason for you to touch this ebuild at all. Rest looks good.